secant/Features/WalletEventsFlow/Views/TransactionDetailView.swift

```diff
@@ -124,7 +124,7 @@ extension TransactionDetailView {
                     Text("reply-to address included")
                     Spacer()
                     Button {
-                        viewStore.send(.replyTo(transaction.address))
+                        viewStore.send(.replyTo(transaction.address.redacted))
```

Comment:
same as above

Reply:
Short answer: no

Long one: it only creates a redacted struct of the string so any attempt to dump it will be redacted. The value inside is still just a String, all unit tests passing. We don't use description of customDump to pass values around, it's only a protection from the logs.